txpool/txpool.go

```diff
@@ -358,25 +367,25 @@ func (p *TxPool) Drop(tx *types.Transaction) {
 
 	// drop promoted
 	dropped := account.promoted.clear()
-	p.index.remove(dropped...)
-	p.gauge.decrease(slotsRequired(dropped...))
+	droppedCount += len(dropped)
```

Comment:
Why not shove this increment logic also in `clearAccountQueue`?

Reply:
[Moved](https://github.com/0xPolygon/polygon-edge/pull/380/commits/6a7ed4f2a87b4f7aad4025a9e921d91552622915)